src/collect.js

```diff
@@ -925,8 +925,7 @@ window.MediathreadCollect = {
         this.displayAsset = function(asset,index) {
             var assetUrl = asset.sources[asset.primary_type];
             if (typeof assetUrl !== 'undefined') {
-                // make sure to strip out any url params
```

Comment:
Yeah, it looks like this was added as part of a fix for the 9/11 archive here:
https://github.com/ccnmtl/SherdJS/commit/da4a0262aee8a7ad4a4bd22f8777ed24e7a78536

Which doesn't seem to be working right now anways:
https://pmt.ccnmtl.columbia.edu/item/98114/